### PR TITLE
Multiplay paths patch

### PR DESCRIPTION
**Pull request by heedson**
The README and test file should refer to the multiplay fork of the statsd client.

On updating of the import to the test file, it was noticed that the previous PR, that the New() function should no longer return a muted client on `UDP no response`.

### CODE REVIEW

File: conn.go [hunk 1]
```diff
@@ -46,18 +46,6 @@ func newConn(conf connConfig, muted bool) (*conn, error) {
 	if err != nil {
 		return c, err
 	}
-	// When using UDP do a quick check to see if something is listening on the
```
Comment (stevenh) [inline conn.go]: This is a duplicate of the previous PR's commit which needs removing.

File: statsd_test.go [hunk 2]
```diff
@@ -364,11 +364,12 @@ func TestUDPNotListening(t *testing.T) {
 	defer func() { dialTimeout = net.DialTimeout }()
 
 	c, err := New()
-	if c == nil || !c.muted {
```
Comment (stevenh) [inline statsd_test.go]: This change should be in a separate commit

**Review by stevenh — CHANGES_REQUESTED** [review]

File: conn.go (hunk 1 above)
Comment (heedson) [reply conn.go]: That is correct. Will rectify.

**Review by stevenh — APPROVED** [review]

**Comment by stevenh** [comment]
Thanks for the PRs 👍
